### PR TITLE
wrap function calls in unbreakable invisible braces

### PR DESCRIPTION
**Pull request by ChristopherChudzicki**
Does what it says, and reorganized javascript tests slightly.

I updated the edge course to see it live... perhaps I should have waited. The uploader script is really great.

### CODE REVIEW

File: mitxgraders-js/MJxPrep.test.js [hunk 1]
```diff
@@ -103,12 +105,24 @@ describe('preProcessEqn', () => {
...
+  } )
+
+} )
+
+describe('wrapVariables', () => {
+
   it('wraps variable names in unbreakable invisible brackets', () => {
     const exprs = [
-      ["f''/f + g_a'+sin(horse)", "{:f'':}/{:f:}+{:g_a':}+sin({:horse:})"],
+      ["f''/f + g_a' + sin(horse)", "{:f'':}/{:f:} + {:g_a':} + sin({:horse:})"],
```
Comment (jolyonb) [inline mitxgraders-js/MJxPrep.test.js]: Wait, how does this work? I thought I stripped spaces out of the entry so that it wouldn't mess with the regex?

Comment (ChristopherChudzicki) [reply mitxgraders-js/MJxPrep.test.js]: Note that I moved your variable wrapping into `wrapVariables`, and this only tests the `wrapVariables` function. So `preprocessEqn` still strips whitespace before wrapping the variables.

That said, what part of `preprocessEqn` requires whistepsace-stripping? Do we really need to strip whitespace? I am not sure...

Comment (jolyonb) [reply mitxgraders-js/MJxPrep.test.js]: How do you think `f (x)` should render? What about `f_{mu nu}?` There are just a bunch of places where a space might be inserted, and I didn't want to deal with this in the regex. Besides, the first thing we do in the processing is to strip spaces... Removing them just made my thought processes much more concrete.

**Comment by jolyonb** [comment]
Merging. Feel free to respond to above comment though. Please run tidy again!

File: mitxgraders-js/MJxPrep.test.js (hunk 1 above)
Comment (ChristopherChudzicki) [reply mitxgraders-js/MJxPrep.test.js]: I think you're right. Stripping whitespace will make `f (x)/2` render correctly, so let's stick with it.

**Pull request merged**